frontend/src/data/nsf23313_2_1/categories/pct.ts

```diff
@@ -0,0 +1,31 @@
+
+export const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'race': new Set<string>([
+        'NativeHawaiianOrOtherPacificIslanderAlone',
+        'Black',
+        'TwoOrMoreRaces',
+        'WhiteAlone',
+        'Asian',
+        'AmericanIndianOrAlaskaNativeAlone',
+        'HispanicOrLatino',
+        'UnknownRace',
+    ]),
+    'education': new Set<string>([
+        'DoctorateDegree',
+        'MastersDegree',
+    ]),
+    'citizenship': new Set<string>([
+        'Citizen',
+        'VisaHolder',
+    ]),
+    'postdoc': new Set<string>([
```

Comment:
`postdoc` or `nonfaculty` dimensions are missing from the dcids files. Are we omitting them from the dcid files?